MetaMask/metamask-extension
# Remove new address alert

Fixes: https://github.com/MetaMask/metamask-extension/issues/14783

## Explanation
See background in issue

## Manual Testing Steps
1. Go through the send flow attempting to send to an account that is not in your address book
2. Ensure the New address detected notification does not show (See videos in Issue)
